src/__tests__/components/Personalisation.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from '@testing-library/react';
+import {
+  Personalisation,
+  PersonalisationType,
+} from '@/src/components/molecules/Personalisation/Personalisation';
+
+const componentProps: PersonalisationType = {
+  isOpen: true,
+};
+
+describe('Personalisation component', () => {
+  it('renders component correctly', async () => {
+    render(<Personalisation />);
```

Comment:
what do you think of snapshot tests?

Reply:
I didn't opt for snapshots because I am only testing that the component is rendering and snapshot would look like an overkill for that since I can test it directly.

Reply:
Fine for now, lets discuss snapshot vs unit test as a team.